draft-ietf-jose-hpke-encrypt.md

```diff
@@ -260,19 +267,22 @@ After verification:
 
 # Key Encryption
 
-HPKE based recipients can be added alongside existing `ECDH-ES+A128KW` or `RSA-OAEP-384` recipients because HPKE is only used to encrypt the content encryption key, and because the protected header used in content encryption is passed to HPKE as Additional Authenticated Data.
-The protected header encoding remains consistent with existing JWE formatting rules.
+Recipients using JOSE-HPKE can be added alongside other recpients (e.g., `ECDH-ES+A128KW` or `RSA-OAEP-384`) as HPKE is used to encrypt the
+Content Encryption Key, which is then processed as specified in JWE.
+
+The protected header used in content encryption is passed to HPKE as Additional Authenticated Data. The protected header encoding remains consistent
+with existing JWE formatting rules.
 
 In HPKE JWE Key Encryption:
 
-- The JWE protected header MUST NOT contain the "alg" when recipients use different algorithms to secure the content encryption key.
-- The JWE protected header SHOULD contain the "alg" when all recipients use the same HPKE algorithm to secure the content encryption key.
-- The recipient unprotected header parameters "psk_id" and "auth_kid" MAY be present.
-- The recipient unprotected header parameter "ek" MUST be present.
-- The recipient unprotected header MUST contain a registered HPKE "alg" value.
+- The Key Management Mode is Key Encryption.
+- When all recipients use the same HPKE algorithm to secure the Content Encryption Key, the JWE Protected Header SHOULD contain "alg". Otherwise
+JWE Protected Header (and JWE Shared Unprotected Header) MUST NOT contain "alg".
```

Comment:
```suggestion
- When all recipients use the same HPKE algorithm to secure the Content Encryption Key, the JWE Protected Header SHOULD contain "alg".
Otherwise, the JWE Protected Header (and JWE Shared Unprotected Header) MUST NOT contain "alg".
```